ntbs-service-unit-tests/DataMigration/NotificationMapperTest.cs

```diff
@@ -109,6 +109,7 @@ public async Task correctlyCreates_basicNotification()
             Assert.Equal("9815779000", notification.PatientDetails.NhsNumber);
             Assert.Equal(new DateTime(1981, 3, 24), notification.PatientDetails.Dob);
             Assert.Equal(2010, notification.PatientDetails.YearOfUkEntry);
+            Assert.Equal(string.Join(" \n", "3 Winglass Place", "Wongaton"), notification.PatientDetails.Address);
```

Comment:
Very minor thing, but the `string.Join` here is duplicating your production code, so reduces the scope of your test. It would be a slight improvement to just have the literal `"3 Winglass Place \nWongaton"` here.